tests/test_confluence_api.py

```diff
@@ -60,74 +86,74 @@ def test__configure_atlassian(self, confluence_client):
             == new_sess.adapters["https://"].total
         )
 
-    def test_find_code_metadata(self, confluence_client):
-        """Test find_code_metadata method with mocked response."""
-
-        def mock_response(url):
-            """Return a mock response object with code block element."""
-
-            class MockResponse:
-                @property
-                def content(self):
-                    if url == "https://example.com/no_code_block":
-                        return b"<div>No code block here</div>"
-                    elif url == "https://example.com/single_code_block":
-                        return b'<code>{"title": "awesome project"}</code>'
-                    elif url == "https://example.com/multiple_code_blocks":
-                        return (
-                            b'<code>{"title": "first project"}</code>'
-                            + b'<code>{"title": "second project"}</code>'
-                        )
-
-            return MockResponse()
-
+    def test_extract_json_metadata(self, confluence_client, response_fixt):
+        """Test extract_json_metadata method with mocked response."""
         # set up
         client = confluence_client
         # single code block should pass -----------------------------------
         # Mock the _get_atlassian_page_content method
-        url = "https://example.com/single_code_block"
+        url = "https://example.com/json_code_block"
         when(client)._get_atlassian_page_content(url).thenReturn(
-            mock_response
+            response_fixt
```

Comment:
Personal preference, but I would probably put this on a single line as it won't overflow 70-90 character limit.

Reply:
I'll have a look at that, it's probably an automated linter thing. I prefer single line where appropriate, too.